jira_docugen.appendix.tex

```diff
@@ -0,0 +1,99 @@
+\scriptsize{
```

Comment:
This is missing a `\section` heading.

Reply:
OK, will update the templates in docsteady.

Reply:
The heading is in the master LDM-533.tex. No need to update the templates.